gpflow/test_util.py

```diff
@@ -107,3 +108,23 @@ def test_context(self, graph=None):
         graph = self.test_graph if graph is None else graph
         with graph.as_default(), self.test_session(graph=graph) as session:
             yield session
+
+
+def is_continuous_integration():  # pragma: no cover
```

Comment:
Isn't it easier and cleaner to write 
```python
is_continous_integration = os.environ.get('CI', False)
```

Reply:
@sanket-kamthe -- I've made your proposed changes

Reply:
No, it's not, just the environment variable existing does not mean that we're in CI - it could be set to "false" or "0". I think explictly checking for true or 1 is safer.

Reply:
I agree but this code already does that if it is set to zero or False it will be zero or False. 
If 'CI' set to False then it should return False. Travis and other environments always have this flag set to `True`.
Check her for details https://docs.travis-ci.com/user/environment-variables/#Default-Environment-Variables

Reply:
No, it doesn't; environment variables are always strings, and "0" and "False" both evaluate to `True`.

Reply:
Anyway, changed it back already :P